lib/coinbase/address.rb

```diff
@@ -122,18 +132,31 @@ def to_s
     # Normalizes the amount of ETH to send based on the asset ID.
     # @param amount [Integer, Float, BigDecimal] The amount to normalize
     # @param asset_id [Symbol] The ID of the Asset being transferred
-    # @return [BigDecimal] The normalized amount in units of ETH
+    # @return [BigDecimal] The normalized amount in units of Wei
```

Comment:
Please update line 49 if the server is returning balances in units of wei

Reply:
The server is returning balances in Wei and they are then normalized to ETH on the SDK side.